gtk4/src/requested_size.rs

```diff
@@ -0,0 +1,28 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
+
+use glib::translate::*;
+
+#[derive(Debug, Copy, Clone)]
+#[repr(transparent)]
```

Comment:
Doesn't have to be transparent unless you want to cast between the pointers of these two types